src/lib.rs

```diff
@@ -269,6 +269,13 @@ impl<F: Read + Seek> CompoundFile<F> {
             );
         }
 
+        if inner_len < header.version.sector_len() as u64 {
+            invalid_data!(
+                "Invalid CFB file (length of {} != header length of {})",
```

Comment:
Should this be "(length of {} < sector length of {})"?

Reply:
Yep, good catch!

Fixed these and the other nits in [496549c](https://github.com/mdsteele/rust-cfb/pull/24/commits/496549c3e8ac034e3da4afb69d4e2eadf077db10).